### DISCUSSION TITLE
textures appear on ios, but textures are not loaded on android

### DISCUSSION THREAD

**Discussion by FehmiCan16** [google/model-viewer#4399] [Q&A]
Hello. I made a configurator application with model viewer. When it loads textures and shows them in AR, textures are loaded in iOS phones, but not in Android. Where could the problem be?

site link:
http://3d.arvithome.com/bufeefesprofil/

**Comment by zjafarpishe** [comment]
use ar-modes="webxr scene-viewer quick-look"

**Comment by elalish** [comment]
Scene Viewer doesn't support updates to GLBs (since it redownloads the original model). WebXR mode is our default, so if you're not getting it then either you're not serving over HTTPS or the permission has been denied. Check your JS console logs.
